app/views/foreman_discovery/layouts/application_react.html.erb

```diff
@@ -0,0 +1,14 @@
+<% content_for(:javascripts) do %>
+  <%= webpacked_plugins_js_for :'foreman_discovery' %>
+<% end %>
+<% content_for(:stylesheets) do %>
+  <%= webpacked_plugins_css_for :'foreman_discovery' %>
+<% end %>
+
+<% content_for(:content) do %>
+  <%= notifications %>
+  <div id="organization-id" data-id="<%= Organization.current.id if Organization.current %>" ></div>
+  <div id="user-id" data-id="<%= User.current.id if User.current %>" ></div>
```

Comment:
No need to bind data into the attributes, current user and taxonomies should be available in store.